python/ray/experimental/state.py

```diff
@@ -16,6 +17,10 @@
 from ray.utils import (decode, binary_to_object_id, binary_to_hex,
                        hex_to_binary)
 
+# These channels must be kept up-to-date with the C++ codebase
+LOCAL_SCHEDULER_INFO_CHANNEL = b"local_schedulers"
+XRAY_HEARTBEAT_CHANNEL = b"6"
```

Comment:
Don't hardcode this, you can use

```python
XRAY_HEARTBEAT_CHANNEL = str(
    ray.gcs_utils.TablePubsub.HEARTBEAT).encode("ascii")
```

also consider moving that into `gcs_utils.py` (it also appears in at least one other place `monitor.py`).